frontend/selectors/index.js

```diff
@@ -143,6 +168,37 @@ export const getRechargeTotalPrice = createSelector(
   }
 );
 
+/**
+ * Gets line items from recharge cart reducer
+ * @returns {Array||null}
+ */
+export const getRechargeCartLineItems = createSelector(
+  getRechargeCartState,
+  ({ line_items }) => {
+    if (!line_items) {
+      return null;
+    }
+
+    return line_items;
+  }
+);
+
+export const getCartItemLineInfo = createSelector(
+  getCartItemProduct,
+  getRechargeCartState,
+  ({ id }, { line_items }) => {
```

Comment:
why not use getRechargeCartLineItems rather than getRechargeCartState?

Reply:
Updated to use getRechargeCartLineItems